### PR TITLE
Update Docker image CE install for licenses

### PR DESCRIPTION
**Pull request by jjerger**
Moves binaries to own folder and places LICENSE.TXT in the same directory, then updates system path to point to the new location

### CODE REVIEW

File: daisy_workflows/image_build/windows_for_containers/container_image_install_ce.ps1 [hunk 1]
```diff
@@ -151,6 +151,19 @@ function Run-FirstBootSteps {
   Write-Host 'Running docker install script'
   $dockerwait = Start-Process -PassThru -FilePath "powershell.exe" -ArgumentList "$env:TEMP\install-docker-ce.ps1"
   Wait-Process -InputObject $dockerwait
+
+  Write-Host 'Copying Docker license information'
+  mkdir C:\Windows\System32\Docker
```
Comment (bkatyl) [inline daisy_workflows/image_build/windows_for_containers/container_image_install_ce.ps1:156]: Just so this doesn't get committed. James and I chatted about a set of changes to this PR.

**Comment by jjerger** [comment]
Should be good now pending Brian's approval.

**Review by bkatyl — APPROVED** [review]

**Pull request merged**